package.json

```diff
@@ -28,13 +28,14 @@
     "nock": "^5.2.1",
     "proxyquire": "^1.7.3",
     "rimraf": "^2.4.3",
+    "sepia": "^2.0.1",
     "snazzy": "^2.0.1",
     "standard": "^5.3.1",
     "tape": "^4.2.2"
   },
   "scripts": {
-    "test": "mocha --recursive",
-    "pretest": "npm run lint",
+    "test": "mocha",
+    "pretest": "npm run lint && rimraf test/integration/node_modules",
```

Comment:
I've found that this improves test reliability and speed considerably on my machine. Calling `rimraf` programatically seems to take 1 minute or so.